webpack.config.js

```diff
@@ -123,6 +126,9 @@ module.exports = {
             // bottom of <head> or the bottom of <body>, and I'm a bit scared
             // about moving them.
             inject: false,
+            vars: {
+                og_image_url: og_image_url,
```

Comment:
Would it make more sense to just configure it here in webpack config instead of using an env. variable?

Reply:
Well, at least this is more of a normal input to the build system. Arguably if people can edit the webpack config they may as well just edit the source html.

Reply:
Fair!